packages/target-graph/src/types/Target.ts

```diff
@@ -42,6 +42,11 @@ export interface Target {
    */
   options?: Record<string, any> /* eslint-disable-line @typescript-eslint/no-explicit-any */;
 
+  /**
+   * Whether the target should be displayed by reporters
```

Comment:
Include an example, not sure what the use case is.

Reply:
(Judging from other changes, it seems like the root node is a "hidden" node to avoid reporting.)

Reply:
It's to accommodate the fact that we actually have a graph with multiple entry points - so the entry points are essentially thrown inside a "hidden" root node. Maybe I could have called it an "isRoot" flag. But it is what it is. It's a hint for reporters to ignore, really.